colossus/src/main/scala/colossus/controller/InputController.scala

```diff
@@ -1,11 +1,14 @@
 package colossus
 package controller
 
+import colossus.controller.OutputResult.Failure
```

Comment:
Personally, I would change this name, so it doesn't clash with a class name that is widely used by Scala.